Tasks/DownloadGitHubReleaseV0/task.loc.json

```diff
@@ -47,9 +47,6 @@
         "specificVersion": "Specific Version",
         "specificTag": "Specific Tag"
       },
-      "properties": {
```

Comment:
Why are we removing editable?

Reply:
input for this field has to be strictly from values in **options**